contributing/documentation/content_guidelines.rst

```diff
@@ -0,0 +1,181 @@
+
+.. _contributing/titles:
+
+Titles
+------
+
+To write a good title :
+
+- **Be concise.**
+- **Avoid sentences**, questions, and titles starting with "how to."
+- **Don't use pronouns** in your titles, especially 2nd person (*your*)
+
+.. _contributing/document-structure:
+
+Document's structure
+====================
+
+Use different **headings levels** to organize your text by sections and sub-sections. Your headings
+are also displayed in a dynamic *navigation bar* on the side.
+
++---------------------------------------------------------------------------------------+
+| | **H1: Page Title**                                                                  |
+| | Your *page title* gives your reader a quick and clear understanding of what your    |
+|   content is about. It is also referenced in the section's *table of contents*.       |
+|                                                                                       |
+| The *content* in this section describes the upcoming content from a **business point  |
+| of view**, and shouldn't put the emphasis on Odoo, as this is documentation and not   |
+| marketing.                                                                            |
+|                                                                                       |
+| Start first with a **Lead Paragraph**, which helps the reader make sure that they've  |
+| found the right page, then explain the **business aspects of this topic** in the      |
+| following paragraphs.                                                                 |
++-----+---------------------------------------------------------------------------------+
+|     | | **H2: Section Title (configuration)**                                         |
+|     | | This first H2 section is about the configuration of the feature, or the       |
+|     |   prerequisites to achieve a specific goal. To add a path, make sure you        |
+|     |   use the ``:menuselection:`` specialized directive (see link below).           |
+|     |                                                                                 |
+|     | | Example:                                                                      |
+|     | | To do so, go to ``:menuselection:`App name --> Menu --> Sub-menu```, and      |
+|     |   enable the XYZ feature.                                                       |
++-----+---------------------------------------------------------------------------------+
+|     | | **H2: Section Title (main sections)**                                         |
+|     | | Create as many main sections as you have actions or features to distinguish.  |
+|     |   The title can start with a verb, but try to avoid using "Create ...".         |
++-----+-----+---------------------------------------------------------------------------+
+|     |     | | **H3: Subsection**                                                      |
```

Comment:
Maybe specify it is possible to have H4, H5, and H6?
Maybe also specify that contributors must avoid weird structures such as H2 -> H4 -> H3 -> H4 -> H2?

Reply:
Actually, it is not possible to have these weird structures as it is explained in the RST cheat sheet.
Also, the other levels are explained there. I just don't have any "instruction" to give about other heading levels.
Do you have any suggestions for what could be said about these other levels?

Cheers :)

Reply:
I don't have anything special to say. I basically think that H1, H2, and H3 are enough to write a good doc (sometimes H4 can be useful, though).
I didn't check the rst cheat sheet so that ok like this :+1: